docs/index.rst

```diff
@@ -93,6 +93,32 @@ Features
 - Integration with most SQL-speaking RDBMS through SQLAlchemy
 - Deep integration with Druid.io
 
+The following RDBMS are currently suppored:
```

Comment:
[optional]: "Other database engines with a proper DBAPI driver and SQLAlchemy dialect should be supported as well."  (with the right links to DBAPI spec and SQLAlchemy definition)

Reply:
@mistercrunch where do you think this logic should reside? In `index.rst` or `README.md`?

Reply:
both? or one could link to the other?